wandb/cli/cli.py

```diff
@@ -921,6 +922,24 @@ def _parse_settings(settings):
             if launch_queue.count(" ") > 1:
                 launch_queue = f"{launch_queue!r}"
 
+        scheduler_config = launch_config.get("scheduler", {})
+        scheduler_entrypoint = [
+            "wandb",
+            "scheduler",
+            "WANDB_SWEEP_ID",
+            "--queue",
+            launch_queue,
+            "--project",
+            project,
+            "--num_workers",
+            config.get("launch", {}).get("num_workers", '2'),
+        ]
+
+        if _job:
+            scheduler_entrypoint += ["--job", _job,]
+        elif _docker_uri:
+            scheduler_entrypoint += ["--docker_image_uri", _docker_uri]
```

Comment:
could add else: warn; I think its actually handled above but could be nice to be extra safe